components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationManagementService.java

```diff
@@ -228,4 +229,17 @@ public abstract String exportSPApplication(String applicationName, boolean expor
      */
     public abstract ImportResponse importSPApplication(SpFileContent spFileContent, String tenantDomain, String
             username, boolean isUpdate) throws IdentityApplicationManagementException;
+
+    /**
+     * Import Service Provider application from stream.
```

Comment:
This comment looks wrong, there is no stream involved (within this method's scope)